package/network/services/samba36/files/samba.init

```diff
@@ -15,6 +14,7 @@ smb_header() {
 		local net
 		for net in $interface; do
 			local device
+			[ "$(network_is_up $net && echo yes)" = "yes" ] || continue
 			network_get_device device "$net" && {
```

Comment:
&& echo yes seems bad. what does network_is_up return on true/false case ? you should test for this. it looks like you are testing the return code here.